overlord/snapstate/snapmgr_test.go

```diff
@@ -788,7 +789,7 @@ func makeTestSnap(c *C, snapYamlContent string) (snapFilePath string) {
 
 func (s *snapmgrTestSuite) TestInstallFirstLocalRunThrough(c *C) {
 	// use the real thing for this one
-	snapstate.MockOpenSnapFile(snapstate.OpenSnapFileImpl)
+	snapstate.MockOpenSnapFile(backend.OpenSnapFile)
```

Comment:
no, because TearDownTest does that (also for these that becomes a noop because this is the original value), maybe obscure, but correct